src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToUseVB.cs

```diff
@@ -115,12 +109,20 @@ private static (VBRuntime, string[]) GetExpectedOutputs(string targetFramework,
             switch ((targetFramework, isExe))
             {
                 case ("net45", true):
-                    return (VBRuntime.Default, new[]
+                    var files = new[]
+                        {
+                            "HelloWorld.exe",
+                            "HelloWorld.exe.config",
+                            "HelloWorld.pdb"
+                        };
+                    if (TestProject.ReferenceAssembliesAreInstalled(TargetDotNetFrameworkVersion.Version471))
                     {
-                        "HelloWorld.exe",
-                        "HelloWorld.exe.config",
-                        "HelloWorld.pdb"
-                    });
+                        return (VBRuntime.Default, files);
```

Comment:
Is there a change in behavior based on whether the reference assemblies are installed?  Is this expected?

Reply:
No change in behavior. I removed the [special case](https://github.com/dotnet/sdk/pull/10981/files/3e7b18b53b58f2a2ae97540d827924fd7986514a#diff-d43ce974025b3d970a2ce51853809514L43) for when reference assemblies weren't installed. The logic is still the same if they are installed (we assert that we're using the [default runtime](https://github.com/dotnet/sdk/pull/10981/files/3e7b18b53b58f2a2ae97540d827924fd7986514a#diff-d43ce974025b3d970a2ce51853809514R120)) but when they aren't installed we now automatically reference them and as a result we use a [referenced runtime](https://github.com/dotnet/sdk/pull/10981/files/3e7b18b53b58f2a2ae97540d827924fd7986514a#diff-d43ce974025b3d970a2ce51853809514R124).